opencti-platform/opencti-front/src/private/components/settings/organizations/SettingsOrganization.tsx

```diff
@@ -0,0 +1,234 @@
+import { graphql, useFragment } from 'react-relay';
```

Comment:
We can't CRUD organization from settings ?

Reply:
It's not part of the MVP even if, I agree, it doesn't take a lot of extra time